### ISSUE TITLE
make error GIT_REVISION

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/AViz#12]
In compiling pr #6,  on some machines there is the following error:

``` 
src/main.cpp:107: error: expected ‘)’ before ‘GIT_REVISION’
```

GIT_REVISION is set in aviz.pro:
```
git describeGIT_REVISION = $$system($$quote(git describe))
DEFINES += $$quote(GIT_REVISION=\'\"$$GIT_REVISION\"\')
```

**Comment by nathanfranklin** [comment]
@jeremy-rutman  can you add the what the OS is and if the machine has git installed.  and what "git describe" returns.

Maybe we should have a fallback for cases where the machine doesnt have git?  

My original goal here was to have these testing versions AVIZ that could display (i.e. in about menu) their Version number and something more specific (i.e. commit number)

**Comment by jeremy-rutman** [comment]
this compile error was seen on redhat  -
uname -a
Linux phony2.technion.ac.il 2.6.32-504.12.2.el6.x86_64 #1 SMP Sun Feb 1 12:14:02 EST 2015 x86_64 x86_64 x86_64 GNU/Linux

The git version is a bit lagging behind (as expected in rhel)
[rutman@phony2 AViz]$ git describe
v6.1-107-g282e954

**Comment by nathanfranklin** [comment]
@jeremy-rutman , i fixed this issue in #24.
